doc-site/docs/components/makefile.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Explaining the Makefile in your repository
+sidebar_label: Makefile Lifecycle
+sidebar_position: 2
+---
+
+## Overview
+The [Makefile] shipped with your repository drives the creation of your infrastructure, it defines the order of execution in its `apply` make target.
+
+The [Makefile] is idempotent by design, meaning when `zero apply` runs it will attempt to reach the same final state every time. Therefore if the script determines a resource has previously been created it will skip the creation when run again, meaning `zero apply` can be re-run after initial setup.
```

Comment:
```suggestion
The [Makefile] is idempotent by design, meaning when it runs it will attempt to reach the same final state every time. Therefore if the script determines a resource has previously been created it will skip the creation when run again, meaning `make apply` can be re-run from inside your project after the initial setup when you ran `zero apply`. You may want to use one of the specific targets below though. For example, if you made a change to your staging infrastructure Terraform you could just run `make apply-env`
```

Reply:
Maybe something like this?

Reply:
yes! and maybe ill add a link to https://getzero.dev/docs/modules/aws-eks-stack/guides/managing-terraform#intended-workflows